substrate/client/rpc/src/lib.rs

```diff
@@ -45,3 +45,147 @@ pub mod testing;
 
 /// Task executor that is being used by RPC subscriptions.
 pub type SubscriptionTaskExecutor = std::sync::Arc<dyn sp_core::traits::SpawnNamed>;
+
+/// JSON-RPC helpers.
+pub mod utils {
```

Comment:
Nit; could be moved to a utils.rs file

Reply:
Also; we could add a couple of tests for this pipe_from_stream behaviour?

Reply:
Done and done